src/app/App.tsx

```diff
@@ -5,11 +5,19 @@ import DetailDogCard from './pages/DetailDogCard/DetailDogCard';
 import ContactForm from './pages/ContactForm/ContactForm';
 import SettingsForm from './pages/SettingsForm/SettingsForm';
 import SearchForm from './pages/SearchForm/SearchForm';
+import BackButton from './components/BackButton/BackButton';
+import styles from './App.module.css';
 
 function App(): JSX.Element {
+  function goBack() {
+    history.back();
+  }
   return (
     <>
       <BurgerMenu />
+      <button type="button" onClick={goBack} className={styles.backButton}>
+        <BackButton />
+      </button>
```

Comment:
Instead of writing the goBack function into the app, you should write it into your BackButton Component that you already have. This way, your BackButton is an actual backbutton 😉
And instead of styling the component in the app, it would be better to create a file BackButton.module.css.